test/integration/features/cmd-image.feature

```diff
@@ -0,0 +1,84 @@
+@cmd-image @command
+Feature: Basic image caching test
+  As a user I am able to import and export container images from a local OCI repository
+  located in the $MINISHIFT_HOME/cache directory
+
+  Scenario: As a user I can export a container image from a running Minishift instance
+    Given Minishift has state "Does Not Exist"
+      And executing "minishift delete --force --clear-cache" succeeds
```

Comment:
+1 Elegant. (We should later probably secure that cache is not persisting via multiple features.)